Gruntfile.js

```diff
@@ -287,10 +287,20 @@ module.exports = function (grunt) {
             all: [
                 '<%= meta.grunt %>',
                 '<%= meta.src %>',
-                '<%= meta.test %>'
+                '<%= meta.test %>',
+                '!src/extensions/default/HTMLHinter/slowparse/**', // Not our code, so just ignore it
+                '!src/extensions/default/HTMLHinter/tooltipsy.source.js', // Klever has no idea how this works, so its ignored
```

Comment:
It looks like all of these (except makedrive-sync-icon) are references to third-party code of some kind. Your first comment should do it.